src/query-store.ts

```diff
@@ -444,7 +444,7 @@ export const useQueryCache = /* @__PURE__ */ defineStore(QUERY_STORE_ID, ({ acti
     return Promise.all(
       getEntries(filters).map((entry) => {
         invalidate(entry)
-        return entry.active && fetch(entry)
+        return entry.active && entry.options?.enabled && fetch(entry)
```

Comment:
Enabled can also be a ref, so we need this

```suggestion
        return entry.active && toValue(entry.options?.enabled) && fetch(entry)
```